site/generatedComponentConfig.js

```diff
@@ -0,0 +1,134 @@
+/* This file is auto-generated by terra-site's generate-component-config config. */
```

Comment:
The file generated by the [`generate-component-config` script](https://github.com/cerner/terra-site/blob/dynamic-site/scripts/generate-component-config/generate-component-config.js) provided in terra-site. This output is used the default search paths provided. This file should never need to be touched.

Reply:
Add inline comment to indicate where this script comes from. per matts request